### ISSUE TITLE
repair request before there's a landlord?

### ISSUE THREAD

**Issue by reviser273** [elmo/upstairs#105]
Just putting a note to remind you that repair requests made before a landlord has signed up will have to trigger an opportunity for user to invite a landlord. And most won't know the landlord's email address.

**Comment by elmo** [comment]
Good point. However, it could be an opportunity to get them to invite their landlord.
